xgcm/test/test_grid.py

```diff
@@ -562,6 +578,11 @@ def test_grid_create(all_datasets):
     ds, periodic, expected = all_datasets
     grid = Grid(ds, periodic=periodic)
     assert grid is not None
+    for ax in grid.axes.values():
+        assert ax.boundary is None
+    grid = Grid(ds, periodic=periodic, boundary="extend")
```

Comment:
Can we parameterize the `boundary` value? And also test a version with dict input?